docs/website/contents/for-developers/PreflightGuide.md

```diff
@@ -0,0 +1,101 @@
+ - Quickly gathering empirical statistics of quantities of interest, like minimum/average/maximum size/validation time/frequency of blocks.
+ - Direct access to any on-chain data, like the current stake distribution or UTxO size.
+
+### Disclaimer
+
+ - The node is currently relatively resource-intensive[^resource-relative], see the minimum system requirements [node release page][].
+    - It uses a lot of memory, and the memory requirements are growing as the chain evolves[^utxo-hd].
+    - It inherently requires a lot of disk space (as of January 2024, 150GB).
+    - During syncing, it is quite CPU-intensive (albeit mostly only on a single core).
+      Once syncing is done, it is very light on CPU usage.
+ - You do not need to run the node at all times.
+   The stored blockchain can also be consumed by other developer tools that are independent of the node.
+ - Syncing can take multiple days (full sync in December 2023 on a decently powerful server: 2 days).
+   Unless your network connection is very slow compared to your CPU, the bottleneck is likely your CPU.
+ - It is safe to cleanly interrupt syncing, eg via <kbd>Ctrl</kbd> + <kbd>C</kbd>.
+   The node is able to gracefully resume on restart.
+
+### Running the node
+
+First, get a [cardano-node][] binary, see the README, eg via Nix, binary from the releases or compiling locally, depending on preferences and context.
+
+A basic invocation looks like this:
+
+```shell
+./result/bin/cardano-node \
+    run \
+    --config /path/to/mainnet-config.yaml \
+    --database-path /path/to/db-mainnet \
+    --topology /path/to/mainnet-topology.json \
+    --host-addr 0.0.0.0 --port 3001 \
+    --socket-path /path/to/node.socket
+```
+
+ - `--config`: Get the default Cardano mainnet configuration file [here][cardano mainnet conf].
+    - The logging system supports human-readable or JSON output (look for `ScText`/`ScJson`).
+      `ScJson` sometimes contains more information, and it can be handy for automatic processing via eg `jq`.
+    - Note that Genesis files are still mutable as long as their corresponding era is still in development (like Conway as of early 2024), so your configuration file might need changes once you update your node.
+ - `--topology`: This file defines how/when the node will connect to other nodes.
+   Get the default [here][cardano mainnet topo].
+
+   While syncing, it makes sense to set the valency, ie the number of nodes sampled from the given DNS name to connect to, to `1` for better performance.[^genesis-syncing]
+
+Make sure to store the log output somewhere.
+
+### Syncing to mainnet
+
+Now you are ready to start syncing to mainnet!
+
+During this long process, you can stare at the log output, and eg look for lines containing `Chain extended` or `AddedToCurrentChain`, which (at least) mention the slot number of the most recently added block.
+Bonus points for a plot of your sync progress over time!
+
+## TODO
```

Comment:
What about hinting at other possible things the readers of this document might do and creating an epic with the items below so that we don't have a TODO lingering on?

Reply:
Opened https://github.com/IntersectMBO/ouroboros-consensus/issues/887 :+1: